shared/message/platform/darwin/DarwinSensors.proto

```diff
@@ -129,21 +292,82 @@ message DarwinSensors {
         Servo headTilt       = 20;
     }
 
-    google.protobuf.Timestamp timestamp       = 1;
-    uint32                    cm740ErrorFlags = 2;
-    LEDPanel                  ledPanel        = 3;
-    HeadLED                   headLED         = 4;
-    EyeLED                    eyeLED          = 5;
-    Buttons                   buttons         = 6;
-    float                     voltage         = 7;
-    Accelerometer             accelerometer   = 8;
-    Gyroscope                 gyroscope       = 9;
-    FSRs                      fsr             = 10;
-    Servos                    servo           = 11;
+    /**
+     * Timestamp when our data was taken
+     */
+    google.protobuf.Timestamp timestamp = 1;
+
+    /**
+     * Holds the error code (if any) from the CM740.
+     */
+    uint32 cm740ErrorFlags = 2;
```

Comment:
Do we know what the possible error values/bits/flags are? Are they the same for the servos above?

Reply:
They are the same as for the servos mostly. Only some of the errors can't occur on the CM740

Reply:
Can we get a link to a reference with possible error codes?